view/rooms/list/ListController.js

```diff
@@ -27,7 +27,6 @@ class ListController extends React.Component {
       tableHeaders,
       tableRows,
     } = this.props;
-    tableRows.sort((a, b) => moment(b.updatedAt) - moment(a.updatedAt));
```

Comment:
why are you removing this?

Reply:
@tausifmuzaffar updatedAt only gets updated when the room is patched. If we leave this in, all the old rooms that become deactivated will jump to the top of the list. Do we just want to order by id (when it was created), or something else?